platform/linux-generic/include/odp_schedule_if.h

```diff
@@ -81,6 +81,7 @@ typedef struct {
 	void (*schedule_resume)(void);
 	void (*schedule_release_atomic)(void);
 	void (*schedule_release_ordered)(void);
+	void (*schedule_release_context)(void);
```

Comment:
Why release context? You're switching locks here, not releasing the ordered context, no? That's a very different semantic. If you release the ordered context you can no longer acquire any ordered lock associated with that context.